x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/cytoscape.tsx

```diff
@@ -144,6 +144,8 @@ export function Cytoscape({
   useEffect(() => {
     if (cy) {
       cy.reset();
+      // Refitting because it's possible the the width/height have changed
+      cy.fit();
```

Comment:
Just to confirm - the map shouldn't recalculate the layout and re-render when the window is resized? It doesn't for me, and want to confirm that is the expected behavior as the comment in the PR description `Update width of the cytoscape map when browser is resizing` initially made me think it would re-layout.

Reply:
This is refitting when the user hits 'reset'. Previously this would zoom to the original zoom & pan even if the layout size has changed.